runtime/runtime.go

```diff
@@ -13,46 +13,50 @@ import (
 type Options struct {
 	ConnectionCacheSize int
 	MetastoreDriver     string
-	MetastoreDSN        string
 	QueryCacheSizeBytes int64
 	AllowHostAccess     bool
 	SafeSourceRefresh   bool
+	// GlobalDrivers are drivers whose handles are shared with all instances
+	GlobalDrivers []*Connector
+	// PrivateDrivers are drivers whose handles are private to an instance
+	PrivateDrivers []*Connector
+}
+
+type Connector struct {
+	Type    string
+	Name    string
+	Configs map[string]string
 }
```

Comment:
Since we anyway have the `runtimev1.Connector` type now, should we just use that and remove this one?